sidechain/chain.json

```diff
@@ -0,0 +1,340 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "sidechain",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://side.one/",
+  "pretty_name": "Side Chain",
+  "chain_type": "cosmos",
+  "chain_id": "sidechain-1",
+  "bech32_prefix": "side",
```

Comment:
It is recommended to register this prefix here: https://github.com/satoshilabs/slips/blob/master/slip-0173.md